packages/yoga/src/Datepicker/web/Calendar.jsx

```diff
@@ -205,19 +205,26 @@ function Calendar({
     return new Date(Date.UTC(year, month, day)).getTime();
   };
 
+  const isSameDate = (date1, date2) =>
```

Comment:
Yo @fabiorgreis !!,

First of all, congratulations for your contribution! 🚀🚀🚀

I would like to leave my suggestion on the naming of `data1` and `data2`.

```suggestion
  const isSameDate = (firstDate, secondDate) =>
```

Reply:
Done